packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -447,9 +445,7 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
               {t("cancel")}
             </Button>
             <Button
-              loading={
-                props.isPending || createInviteMutation.isPending || checkIfMembershipExistsMutation.isPending
-              }
+              loading={props.isPending || createInviteMutation.isPending}
```

Comment:
Shouldn't the button be in a `loading` state also when the controller is invoking `checkIfMembershipExists` ?